ipn/ipnlocal/local.go

```diff
@@ -7456,3 +7467,41 @@ func maybeUsernameOf(actor ipnauth.Actor) string {
 	}
 	return username
 }
+
+// VIPServices returns the list of tailnet services that this node
+// is serving as a destination for.
+func (b *LocalBackend) VIPServices() []*tailcfg.VIPService {
```

Comment:
who owns this returned memory?

Or remove the question by returning a view?

Reply:
The memory is owned by the caller. Would you prefer a comment to that effect or a `VIPServiceView`?

Reply:
Documenting it is fine.

If it ever becomes hot/garbage-heavy, then we can viewify it + cache more.